backend/DB/api/static/graph_display.js

```diff
@@ -2,11 +2,11 @@ let width = 1500, height = 750;
 
 let radius = width / 50;
 
-d3.json("api/user/all", (users) => {
+d3.json("/user/all?token=TEST", (users) => {
```

Comment:
Is this supposed to be `token=TEST`, or is this just temporary?

Reply:
This is temporary, as we will get the actual token from elsewhere